### ISSUE TITLE
Read receipt of person on ignore list shouldn't be shown

### ISSUE THREAD

**Issue by Bubu** [vector-im/element-android#2965]
**Describe the bug**
If I ignore a person, that persons RR is still shown along the timeline. Element-web doesn't do this.

**To Reproduce**
* Ignore a person
* Have that person actively read a chat you are in
* See the RR next to new messages

**Expected behavior**
RR of ignored person is not shown.

**Additional context**
 - App version and store: 1.1.0-F
 - Homeserver: bubu1.eu

**Comment by Bubu** [comment]
It also shows typing notifs of ignored users. :-(

**Comment by bmarty** [comment]
According to [the spec](https://matrix.org/docs/spec/client_server/r0.6.1#id144), it's not clear to me if read-receipt and typing notification (so ephemeral events) have to be filtered out by the server or by the client.

I would argue that it could have more sense that the server filters them out. The less the clients have to compute things, the better the perf.

@anoadragon453 what do you think about ephemeral event filtering for ignored users?

Also there are other problems to fix about ignoring users, with higher priority: https://github.com/vector-im/element-android/labels/feature%3Aignored_users

**Comment by Bubu** [comment]
Read-receipts are PDUs though afaik?

**Comment by Bubu** [comment]
Updated the issue with the information that ele-web also suffers from the typing notif problem, even though it *does* do the RR one correctly.

**Comment by bmarty** [comment]
Yes this is https://github.com/vector-im/element-web/issues/6552
